vars/main.yml

```diff
@@ -6,3 +6,9 @@ ansible_distro_major_version: "{{ ansible_distribution_major_version|lower|repla
 
 _sshd_principals: "{{ sshd_principals_default | combine(sshd_principals, recursive=True, list_merge=sshd_principals_list_merge) }}"
 _sshd_config: "{{ sshd_config_default | combine(sshd_config, recursive=True, list_merge=sshd_config_list_merge) }}"
+
+_sshd_banner_file_path: /etc/banner
+_sshd_banner_state: "{{ 'present' if ssh_banner_template else 'absent' }}"
+_sshd_banner_lookup_method: "{{ 'url' if sshd_banner_template is match('http(s)?:\/\/') else 'template' }}"
+_sshd_banner_template_content: "{{ lookup(_sshd_banner_lookup_method, sshd_banner_template, split_lines=False, username=sshd_banner_template_username, password=sshd_banner_template_password) }}"
+_sshd_config_file_path: /etc/ssh/sshd_config
```

Comment:
This variable is not used

Use it in https://github.com/claranet/ansible-role-sshd/blob/3bae04950707d53817f67051c18613e821488e75/tasks/main.yml#L69 or delete it